helpers.js

```diff
@@ -92,27 +92,21 @@ function fixClass(cls) {
   cls = cls.replace(/^(\.)/, "");
   // make other dots safe
   cls = cls.replace(/\\\./g, ".");
-  // remove extras at end
+  // remove > anything
+  cls = cls.replace(/\s?>\s?.*/, "");
+  // remove pseudo-elements (::)
+  cls = cls.replace(/::.*$/, "");
+  // remove pseudo-classes (:)
   cls = cls.replace(
-    /:(focus-within|first-child|last-child|odd|even|hover|focus|active|visited|disabled)$/,
+    /(:(active|after|before|checked|disabled|focus|focus-within|hover|visited|nth-child\((even|odd)\)|(first|last)-child))+$/,
```

Comment:
One rule to cover all known pseudo-classes so that we can look for one or more of them.